health-sdk/sdk/src/main/java/net/gini/android/health/sdk/review/openWith/OpenWithBottomSheet.kt

```diff
@@ -0,0 +1,106 @@
+package net.gini.android.health.sdk.review.openWith
+
+import android.content.Intent
+import android.graphics.Typeface
+import android.net.Uri
+import android.os.Bundle
+import android.text.Html
+import android.text.SpannableString
+import android.text.SpannableStringBuilder
+import android.text.Spanned
+import android.text.SpannedString
+import android.text.TextUtils
+import android.text.method.LinkMovementMethod
+import android.text.style.ClickableSpan
+import android.text.style.ForegroundColorSpan
+import android.text.style.StyleSpan
+import android.text.style.UnderlineSpan
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.core.text.buildSpannedString
+import androidx.fragment.app.viewModels
+import net.gini.android.health.sdk.R
+import net.gini.android.health.sdk.databinding.GhsBottomSheetOpenWithBinding
+import net.gini.android.health.sdk.paymentprovider.PaymentProviderApp
+import net.gini.android.health.sdk.util.GhsBottomSheetDialogFragment
+import net.gini.android.health.sdk.util.autoCleared
+import net.gini.android.health.sdk.util.setBackgroundTint
+
+/**
+ * Interface for forwarding the request to share a PDF document
+ */
+internal interface OpenWithForwardInterface {
```

Comment:
[Request] For listener interfaces we should use `Listener` as the suffix: `OpenWithForwardListener`.